java/android/Portfile

```diff
@@ -3,46 +3,59 @@
 PortSystem 1.0
 
 name                android
-version             23
+version             28.0.3
 categories          java devel emulators
-maintainers         nomaintainer
+maintainers         openmaintainer krischik
```

Comment:
By convention, `openmaintainer` is always last. So change this to:

`maintainers        krischik openmaintainer
`

Reply:
He's not a Macports member so shouldn't it be

`maintainers {@krischik bar.com:foo} openmaintainer`

??

Reply:
Martin is "retired" [according the developers page](https://trac.macports.org/wiki/MacPortsDevelopers). I guess that can change now :). But it wouldn't hurt to add the GitHub handle too.

Reply:
`maintainers {krischik @krischik} openmaintainer`

Reply:
Gotcha. So he's unretiring. In which case he should be given commit access too.

Reply:
@krischik: You still have commit access right?

Reply:
I'm pretty sure it would say "member" above if he did.